fronts-client/src/reducers/frontsReducer.ts

```diff
@@ -63,6 +63,20 @@ const reducer = (
         newState
       );
     }
+    case 'FETCH_FRONT_REMOVE_COLLECTION_SUCCESS': {
```

Comment:
I can't see that this action is dispatched anywhere! Do we really need this case (and the above case), and the accompanying action types in `types/Actions.ts`?

Reply:
Yeah, infact that was my question i wanted to check with you as I didnt spot this action anywhere during running on browser. Thanks for confirming I will remove them 👍